src/components/Territory.vue

```diff
@@ -23,7 +23,7 @@
               </b-button>
             </b-button-group>
             <b-button-group v-if="viewMode==='address-list'" size="sm">
-              <b-button v-if="canAdmin" variant="danger" @click="resetNH(true)">Reset</b-button>
+              <b-button v-if="canAdmin" variant="danger" @click="resetNH(true)">Check In</b-button>
```

Comment:
Right now, only admins can see this button. We're gonna open it up to everyone... with some auth checks:

1. the current user must be the publisher who checked out the territory

2. Or, the current user must have "canWrite" privileges (see auth.js getters).